src/components/personal-info.jsx

```diff
@@ -11,6 +11,19 @@ function PersonalInfo() {
     setUserInfo({...userInfo, [e.target.name]: e.target.value})
   }
 
+const handleSubmit = async (e) => {
+    e.preventDefault();
+
+    try {
+      const response = await axios.post("/resume/personal", //update the route after it's implemented in the backend
```

Comment:
what does axios.post here?

Reply:
> Axios is a [promise-based](https://javascript.info/promise-basics) HTTP Client for [node.js](https://nodejs.org/) and the browser. It is [isomorphic](https://www.lullabot.com/articles/what-is-an-isomorphic-application) (= it can run in the browser and nodejs with the same codebase). On the server-side it uses the native node.js http module, while on the client (browser) it uses XMLHttpRequests.

So in other words, it's just a fancy way of sending `POST` requests (to the backend). We could have used `fetch` too.

After we are done with the development with the backend, we will run
```bash
yarn build
```

which will provide the html, css and js files which will be served by the backend.